apps/native/app/src/utils/get-organization-slug-from-error.ts

```diff
@@ -0,0 +1,36 @@
+import { ApolloError } from '@apollo/client'
+
+type PartialProblem = {
+  organizationSlug?: string
+}
+
+type CustomExtension = {
+  code: string
+  problem?: PartialProblem
+  exception?: {
+    problem?: PartialProblem
+  }
+}
+
+/**
+ * Extracts the organization slug from the Apollo error, if it exists.
+ */
+export function getOrganizationSlugFromError(error: ApolloError | unknown) {
```

Comment:
Did you try to import this from here, since this is the same?

`libs/react-spa/shared/src/utils/getOrganizationSlugFromError.ts`

Reply:
There is nothing in the app imported from these folders so I don't think it will work. I will try though